packages/template-retail-react-app/app/utils/url.js

```diff
@@ -161,15 +159,14 @@ export const getPathWithLocale = (shortCode, buildUrl, opts = {}) => {
     }
 
     const site = getSiteByReference(siteRef)
```

Comment:
NOTE: This function will return the default site if none is found via the reference value, so we probably didn't need that reference to the default site anyway.